sycl/doc/extensions/MemChannel/MemChannel.asciidoc

```diff
@@ -1,4 +1,4 @@
-= SYCL_EXT_INTEL_MEM_CHANNEL_PROPERTY
+= SYCL_INTEL_mem_channel_property
```

Comment:
Why did you rolled back changes for `SYCL_INTEL_mem_channel_property`?

Reply:
I completely reverted all changes for this extension due to discussion https://github.com/intel/llvm/pull/4432#discussion_r703050909
This extension should not be aligned with SYCL 2020 and the plan for it is the following: https://github.com/intel/llvm/pull/4432#discussion_r702127071

Reply:
I was also not expecting this to be rolled back.  See my other comment.

Reply:
Returned these changes back: https://github.com/intel/llvm/pull/4432/commits/198aacaf7a134a0979f2147d2b5711747593e618

Reply:
Thanks!